test/AWS_Tests/src/S3_Spec.enso

```diff
@@ -261,6 +261,12 @@ add_specs suite_builder =
                 r3.should_be_a Vector
                 r3.map .name . should_contain object_name
 
+        group_builder.specify "list should sort its output" <|
+            r = root.list
+            r.should_be_a Vector
+            r . should_equal (r.sort on=.s3_path)
+            r . should_equal (r.sort on=(x-> x.s3_path.to_text))
```

Comment:
Same here, `.s3_path` is private and implementation detail that should be a black box in the tests, use `path` that returns Text.

Reply:
Changed to `.path`